fastlane/Fastfile

```diff
@@ -58,6 +58,40 @@ platform :ios do
   end
 
 
+  desc 'Run ui tests'
+  lane :uitest do
```

Comment:
This looks like an identical copy-paste of the `:test` lane. Except the value passed for the `:scheme` parameter. Could we instead make both of these use a shared lane that takes a `:scheme` parameter?

See https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Advanced.md#passing-parameters for more info on passing parameters to lanes.

---

Maybe something like:
```ruby
lane :test do
  run_tests(scheme: 'HubFramework', output_label: 'tests')
end

lane :test_ui do
  run_tests(scheme: 'HubFrameworkDemo', output_label: 'ui-tests')
end

private_lane :run_tests do |options|
  scheme = options[:scheme]
  output_label = options[:output_label]

  UI.user_error!("No Xcode scheme provided") unless scheme
  UI.user_error!("No label for the output was provided") unless output_label

  system('pkill', '-9', '-x', 'Simulator')
  UI.success('Simulator app killed')

  device = ENV['TEST_DEVICE_NAME'] || 'iPhone 6s'
  UI.message("Simulator testing device set to '#{device}'")

  output_directory = "build/#{output_label}"
  ENV['XCPRETTY_JSON_FILE_OUTPUT'] = "#{output_directory}/summary.json"

  scan(
    :workspace => 'HubFramework.xcworkspace',
    :scheme => scheme,
    :devices => [device],
    :derived_data_path => 'build/DerivedData',
    :output_directory => output_directory,
    :buildlog_path => 'build/logs/tests',
    :code_coverage => true,
    :output_types => '',
    :fail_build => false,
    :formatter => 'xcpretty-json-formatter'
  )

  trainer(
    path: 'build',
    output_directory: output_directory,
    extension: '.junit'
  )

  codecov() if ENV['CODECOV_TOKEN'] or ENV['TRAVIS'] == 'true'
end
```